extensions/query-store/src/settings/configureDialog.ts

```diff
@@ -84,59 +81,80 @@ export class ConfigureDialog {
 		return true;
 	}
 
-	public async openDialog(): Promise<void> {
-		await this.initializeDialog();
+	public async openDialog(configComponentsInfo: ConfigComponentsInfo[]): Promise<void> {
```

Comment:
add doc comment and mention if the order of the components in the array will be the order they're added to the dialog